src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -124,6 +125,14 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
 
             var antiforgeryDefault = true;
+            if (output.Attributes.ContainsName("method"))
```

Comment:
Yeah, I didn't bind the `method` attribute because `DefaultHtmlGenerator` didn't need it. If we need it now, just bind it.